server/util.go

```diff
@@ -233,6 +233,38 @@ func UpgradeOldPrivValFile(config *cfg.Config) {
 	}
 }
 
+// NewCompletionCmd builds a cobra.Command that generate bash completion
```

Comment:
Maybe this wants to live somewhere in the `client` folder? We need to do some thinking on CLI code organization.

Reply:
done